sqldelight-gradle-plugin/build.gradle

```diff
@@ -46,6 +46,11 @@ dependencies {
 }
 
 test {
+  inputs.files("../gradle/dependencies.gradle").withPathSensitivity(PathSensitivity.RELATIVE)
```

Comment:
Relative path sensitivity only makes sense for directories. What you want here is NONE, because you don't care about the path at all, only the contents.